src/test/scala/tv/teads/wiremock/extension/CalculatorSpec.scala

```diff
@@ -0,0 +1,51 @@
+package tv.teads.wiremock.extension
+
+import java.util.UUID
+
+import com.github.tomakehurst.wiremock.client.WireMock._
+import com.github.tomakehurst.wiremock.extension.ResponseTransformer
+import com.ning.http.client.Response
+import dispatch.{Future, Http, url}
+
+import scala.concurrent.ExecutionContext.Implicits.global
+
+class CalculatorSpec extends ExtensionSpec {
+
+  override val extensions: List[ResponseTransformer] = List(new Calculator)
+
+  val requests: List[(String, String)] = List(
+    (s"$${1+2}", "3"),
+    (s"$${1-2}", "-1"),
+    (s"$${1*2}", "2"),
+    (s"$${1/2}", "0.5"),
+    (s"$${1 + 2}", "3"),
+    (s"$${1.1 + 2.2}", "3.3")
```

Comment:
Add a case with an error (for example with a division by 0) to be sure that it returns the JSON with a non interpreted value instead of throwing an error and return a 404